test/libdevcore/rlp.cpp

```diff
@@ -40,8 +40,86 @@ namespace js = json_spirit;
 namespace dev
 {
 	namespace test
-	{
-		static void buildRLP(js::mValue& _v, RLPStream& _rlp)
+	{	
+		void buildRLP(js::mValue& _v, RLPStream& _rlp);
+		void checkRLPAgainstJson(js::mValue& v, RLP& u);
+
+		void doRlpTests(json_spirit::mValue& v, bool _fillin)
```

Comment:
the fillin option works just as in other suites. if test provided with only 'out' section declared and fillin option set to true, It would try to create RLP class from the 'out' field. If succeed the 'in' field would be filled as VALID, if not - as INVALID.  

I would then generate random RLP byte sequences to see if they are correct with fillin option.